framework/auth/__init__.py

```diff
@@ -48,6 +49,7 @@ def logout():
             del session.data[key]
         except KeyError:
             pass
+    Session.remove(Q('_id', 'eq', session._id))
```

Comment:
I'm not sure if Alex will be available for the next few days. I recall he hashed out some ideas with @lyndsysimon on this before trying this solution. I can say that the problem is that, when logging out, with the new cookie structure the existing log out did nothing at all. It just left you logged in. 

That being said, is there a reason to leave the session in the database? You won't have a logout button unless you're logged in, so you wouldn't be removing an anonymous session. And wasn't there a problem where we had a year and a half of old sessions cluttering our database?